modules/passkey/src/utils/helper.ts

```diff
@@ -0,0 +1,207 @@
+/**
```

Comment:
I don't think we need to just copy-paste the `ethers` code in here:
- We can use stricter types (like `bigint` and `Uint8Array`) in the helper methods, instead of converting things to and from hex for example.
- I don't think we should implement `encodeABI`. We should just manually encode the things in the one place we use it (we have deterministic types, and do not need a general ABI encoding function).
- Buffer isn't supported in the browser, so maybe we shouldn't use it here - and if we do, I don't think we should encode as base64 just to manually convert it to base64url when it supports base64url encoding directly.

Reply:
Done 👍🏾